core/src/lib.rs

```diff
@@ -550,7 +534,7 @@ pub fn rename_file(
         }
         DocumentRenameError::FileNameNotAvailable => UiError(RenameFileError::FileNameNotAvailable),
         DocumentRenameError::CannotRenameRoot => UiError(RenameFileError::CannotRenameRoot),
-        DocumentRenameError::DbError(_) | DocumentRenameError::FailedToRecordChange(_) => {
+        _ => {
```

Comment:
is that... a wildcard match arm? who are you and what happened to the real parth

Reply:
haha I'm going to make this the norm in a future PR